store/mongo/datastore_mongo.go

```diff
@@ -42,7 +42,7 @@ import (
 //       That is, s/DbDevGroup/DbDevAttributesGroupName/
 
 const (
-	DbVersion = "0.2.0"
+	DbVersion = "0.3.0"
```

Comment:
do you think it is a change big enough to make 1.0.0? I think it is.

Reply:
I'm not sure, depends on whether we get rid of the previous version of the document (currently overlapping). If the two versions of the documents remain compatible I think that's minor version material (no strong opinion though).